dask_cloudprovider/gcp/instances.py

```diff
@@ -589,6 +598,7 @@ def __init__(
             "machine_type": self.machine_type,
             "ngpus": ngpus or self.config.get("ngpus"),
             "network": network or self.config.get("network"),
+            "network_projectid": network_projectid,
```

Comment:
Great to see this added as a kwarg. Could you also load from the config too?

```suggestion
            "network_projectid": network_projectid or self.config.get("network_projectid"),
```

You would also need to add a sensible default (probably `null`) to [cloudprovider.yaml](https://github.com/dask/dask-cloudprovider/blob/main/dask_cloudprovider/cloudprovider.yaml).

Reply:
@jacobtomlinson Thanks for the feedback. I've updated the cloudprovider.yaml and code so that the value is loaded when available. I took your suggestion to use `null` as the default value in the YAML file.